keyboards/mectechpad/readme.md

```diff
@@ -0,0 +1,7 @@
+# Mectechpad
```

Comment:
This still needs to match the provided [template](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md).